Fabric.Authorization.Domain/Validators/GroupValidator.cs

```diff
@@ -23,10 +23,10 @@ private void ConfigureRules()
                 .WithMessage("Please specify a Name for this Group.")
                 .WithState(g => ValidationEnums.ValidationState.MissingRequiredField);
 
-            RuleFor(group => group.IdentityProvider)
+            /*RuleFor(group => group.IdentityProvider)
```

Comment:
Is this meant to be here ?

Reply:
This is temporarily commented out for the same reason as the `appsettings.json` default IdP setting.